infra/cifuzz/cifuzz_test.py

```diff
@@ -424,5 +484,57 @@ def test_both_fuzzers_kept_fuzzer(self):
         self.assertEqual(2, len(os.listdir(tmp_dir)))
 
 
+class IsProjectSanitizerUnitTest(unittest.TestCase):
+  """Class to test the is_project_sanitizer function in the cifuzz module.
+    Note: This test relies on the curl project being an OSS-Fuzz project.
+  """
+
+  def test_valid_project_curl(self):
+    """Test if sanitizers can be detected from project.yaml"""
+    self.assertTrue(cifuzz.is_project_sanitizer('memory', 'curl'))
+    self.assertTrue(cifuzz.is_project_sanitizer('address', 'curl'))
+    self.assertTrue(cifuzz.is_project_sanitizer('undefined', 'curl'))
+    self.assertFalse(cifuzz.is_project_sanitizer('not-a-san', 'curl'))
+
+  def test_valid_project_example(self):
+    """Test if sanitizers can be detected from project.yaml"""
+    self.assertFalse(cifuzz.is_project_sanitizer('memory', 'example'))
+    self.assertFalse(cifuzz.is_project_sanitizer('address', 'example'))
+    self.assertFalse(cifuzz.is_project_sanitizer('undefined', 'example'))
+    self.assertFalse(cifuzz.is_project_sanitizer('not-a-san', 'example'))
+
+  def test_invalid_project(self):
+    """Tests that invalid projects return false."""
+    self.assertFalse(cifuzz.is_project_sanitizer('memory', 'notaproj'))
+    self.assertFalse(cifuzz.is_project_sanitizer('address', 'notaproj'))
+    self.assertFalse(cifuzz.is_project_sanitizer('undefined', 'notaproj'))
+
+
+@unittest.skip('Test is too long to be run with presubmit.')
```

Comment:
Test needed to be skipped because it was tripling the run time of the presubmit.

Reply:
How long did it take because of the test?

Reply:
Takes around 300 seconds and there are 2 of them.